### ISSUE TITLE
Try other classification networks as feature extractors   

### ISSUE THREAD

**Issue by CODEC-ZLY** [marco-rudolph/cs-flow#20]
Dear Marco-Rudolph, we are currently working on your research. We explored that when other classification networks were used as feature extractors, the losses sent into CS-flow training became extremely large and could not be well converged. Even when it is used in EfficientNet_V2 - m, also cannot achieve EfficientNet_b5 as feature extractor, network achieved better effect. This question has puzzled me for a long time, and I want to know whether you have made a special optimization for EfficientNet_b5. Theoretically, switching to another pre-trained classification network can also achieve a good effect. If you are busy reading this information,I am particularly looking forward to your reply to my questions, which will benefit me a lot.
  Thank you so much！！！

**Comment by marco-rudolph** [comment]
Hello,
there was not made a special optimization for B5. I use the 'original' pretrained weights from ImageNet provided by pytorch.
It's also not clear to me in part why there are such big differences between the networks. However, I could not observe this effect of a very large loss when using 'use_gamma=True'.
